CONTRIBUTING.md

```diff
@@ -364,6 +364,12 @@ To check for style violations, run:
 make markdownlint
 ```
 
+To update the toc, run:
+
+```bash
+make markdown-toc
```

Comment:
since this is part of `make fix`, I'd suggest treating it similar to "Misspell check", and adding a new section for "Update the tables of content" under "Misspell check" above, and adding a link to it from the `make fix` section above

Reply:
Done, thanks @trask